unstructured/partition/html.py

```diff
@@ -112,7 +116,8 @@ def partition_html(
     return document_to_element_list(
         document,
         include_page_breaks=include_page_breaks,
-        last_modification_date=metadata_last_modified or last_modification_date,
+        last_modification_date=metadata_date or last_modification_date,
```

Comment:
This should stay as it was, #997 changed `metadata_date` to `metadata_last_modified`.

Reply:
Yah, I'm going through and fixing that with the rebase--that happened because of a recent commit